astroid/manager.py

```diff
@@ -361,6 +363,15 @@ def bootstrap(self):
         raw_building._astroid_bootstrapping()
 
     def clear_cache(self):
-        """Clear the underlying cache. Also bootstraps the builtins module."""
+        """Clear the underlying cache. Also bootstraps the builtins module and
+        re-registers transforms."""
         self.astroid_cache.clear()
+        AstroidManager.brain["_transform"] = TransformVisitor()
         self.bootstrap()
+
+        # Load brain plugins: currently done in astroid.__init__.py
+        for module in BRAIN_MODULES_DIRECTORY.iterdir():
```

Comment:
Should we make this a function?

Reply:
I wondered that also. I don't know that we need to expose it publicly. I don't know that we would be able to reuse it in `__init__.py` because I suspect that needs to stay as `import_module` rather than `exec_module`.